doc: Create new chapter about the Dashboard's landing page (#36222)
ceph/ceph: doc/mgr/dashboard.rst

  adjust the level of backfilling activity. List all disks associated with an
  OSD. Set and change the device class of an OSD, display and sort OSDs by
  device class. Deploy new OSDs on new disks/hosts.
* **Hosts**: Display a list of all hosts associated to the cluster, which
  disks are attached, which services are running and which version of Ceph is
  installed.
* **Object Gateway**: List all active object gateways and their performance
  counters. Display and manage (add/edit/delete) object gateway users and their
  details (e.g. quotas) as well as the users' buckets and their details (e.g.
  placement targets, owner, quotas, versioning, multi-factor authentication).
  See :ref:`dashboard-enabling-object-gateway` for configuration instructions.
* **iSCSI**: List all hosts that run the TCMU runner service, display all
  images and their performance characteristics (read/write ops, traffic).
  Create, modify and delete iSCSI targets (via ``ceph-iscsi``). Display the
  iSCSI gateway status on the landing page and info about active initiators.
  See :ref:`dashboard-iscsi-management` for instructions on how to configure
  this feature.

Performance
"""""""""""

> * **Client iOPS**:  Display an overview of Input/Output Operations Per Second on

Reviewer: ```suggestion
* **Client IOPS**:  Display an overview of input and output operations per second on
```

  storage devices the cluster is doing on the client side.
* **Client Throoughput**: Displays the data transfer rate to and from
  the storage media in megabytes per second.

> * **CLient READ/Write**: Displays an overview of Read/Write operations performed

Reviewer: ```suggestion
* **CLient Read/Write**: Displays an overview of read and write operations performed
```

Reviewer 2: Also note the capital "L" in "CLient"

* **Ceph Manager Modules**: Enable and disable all Ceph Manager modules, change
  the module-specific configuration settings.

> Dashboard Landing Page

Reviewer 2: How about:
```suggestion
Overview of the Dashboard Landing Page
```

^^^^^^^^^^^^^^^^^^^^^^
Displays overall cluster status, performance and capacity metrics. Gives instant

> feedback to changes in cluster and provides easy access to subpages of dashboard.

Reviewer 2: ```suggestion
feedback to changes in the cluster and provides easy access to subpages of the dashboard.
```

Status
""""""

* **Cluster Status**: Displays overall cluster health. In case of any error it

>   displays a short description of error and provides a link to the logs.

Reviewer 2: ```suggestion
  displays a short description of the error and provides a link to the logs.
```

* **Hosts**: Display the total number of hosts associated to the cluster and

>   link to subpage displaying the list and description of all the hosts.

Reviewer 2: ```suggestion
  link to the subpage displaying the list and descriptions of all hosts.
```

> * **Monitors**: Displys total number of all MONs and their quorum status, open
>   sessions and link to the subpage having the lists and description of all the

Reviewer 2: ```suggestion
* **Monitors**: Displays the total number of all MONs and their quorum status, open
  sessions and links to the subpage providing a list and descriptions of all
```

  MONs.
* **OSDs**: Displays total number of object storage daemons for Ceph (ceph-osds)
  and the total number of OSDs running (up), total number of OSDs in the cluster
  (in) and total number of OSDs out of the cluster (out). Provides link to the

>   subpage having list of all the OSDs.

Reviewer 2: ```suggestion
  subpage containing a list of all OSDs and related management actions.
```

* **Managers**: Displays the total number of active and standby Ceph Manager

>   daemon (ceph-mgr) running alongside monitor daemons,

Reviewer 2: ```suggestion
  daemons (ceph-mgr) running alongside monitor daemons.
```

> * **Object Gateway**: Displays total number of active object gateways and 
>   provides a link to the subpage displaying list of all the daemons.

Reviewer 2: ```suggestion
* **Object Gateway**: Displays the total number of active object gateways and 
  provides a link to the subpage displaying a list of all object gateway daemons.
```

* **Metadata Servers**: Displays total number of active and standby metadata

>   servers daemons (ceph-mds).

Reviewer 2: I'd suggest to add a reference to "CephFS" here:
```suggestion
  servers daemons for CephFS (ceph-mds).
```

> * **iSCSI Gateways**: Display the total number of iSCSI gateway available, total

Reviewer 2: ```suggestion
* **iSCSI Gateways**: Display the total number of iSCSI gateways available, total
```

  number of active iSCSI gateways (up) and total number of inactive iSCSI

>   Gateways (down). Provides link to the subpage having list of all the iSCSI

Reviewer 2: ```suggestion
  Gateways (down). Provides link to the subpage providing a list of all iSCSI
```

  Gateways.

Capacity
""""""""

* **Raw Capacity**: Displays the total physical capacity used out of the total

>   physical capacity provided to the ceph storage nodes (OSDs).

Reviewer 2: ```suggestion
  physical capacity provided by the ceph storage nodes (OSDs).
```

* **Objects**: Displays the number of objects in use and the status of objects
  including the percentage of healthy, misplaced, degraded and unfound status of

>   objects. Object is the smallest unit of data storage in ceph cluster.

Reviewer 2: ```suggestion
  objects. An object is the smallest unit of data storage in a Ceph cluster.
```

* **PG Status**: Displays the total number of placement groups in use and the

>   status of the placement group, including the percentage of clean, working,

Reviewer 2: ```suggestion
  status of placement groups, including the percentage of clean, working,
```

  warning and unknown status of placement groups.
* **Pools**: Displays the total number of pools and links to the subpage that
  lists all Ceph pools and their details.

> * **PGs per OSD**: Displays the number of placement groups per object stotage

Reviewer 2: ```suggestion
* **PGs per OSD**: Displays the number of placement groups per object storage
```

  daemons.

Performance
"""""""""""

> * **Client READ/Write**: Displays an overview of rate of read and write operations,

Reviewer 2: ```suggestion
* **Client READ/Write**: Displays an overview of the rate of read/write operations,
```

  the total number of input and output operations performed by the cluster on
  storage devices on the client side.

> * **Client Throughput**: Displays the data transfer rate to and from the storage
>   media.

Reviewer 2: ```suggestion
* **Client Throughput**: Displays the data transfer rate to and from Ceph clients.
```

* **Recovery throughput**: Displays rate of moving the data back to the cluster
  if the cluster is recovering to parity when the hard drive is lost.
* **Scrubbing**: Displays whether Ceph is comparing the data to other pieces of

>   data to ensure there is no corruption on cluster

Reviewer 2: ```suggestion
  data to ensure there is no data corruption on the cluster's storage devices.
```

* **Ceph Manager Modules**: Enable and disable all Ceph Manager modules, change
  the module-specific configuration settings.

Overview of the Dashboard Landing Page
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Displays overall cluster status, performance and capacity metrics. Gives instant
feedback to changes in the cluster and provides easy access to subpages of the
dashboard.

Status
""""""

* **Cluster Status**: Displays overall cluster health. In case of any error it
  displays a short description of the error and provides a link to the logs.
* **Hosts**: Displays the total number of hosts associated to the cluster and

>   link to subpage displaying the list and descriptions of all hosts.

Reviewer 3: ```suggestion
  links to the subpage displaying the list and descriptions of all hosts.
```

* **Ceph Manager Modules**: Enable and disable all Ceph Manager modules, change
  the module-specific configuration settings.

Dashboard Landing Page
^^^^^^^^^^^^^^^^^^^^^^
Displays overall cluster status, performance and capacity metrics. Gives instant
feedback to changes in cluster and provides easy access to sub pages of dashboard.

Status
""""""

* **Cluster Status**: Displays overall cluster health. 

> * **Hosts**: Display a list of all hosts associated to the cluster, which

Reviewer 2: I'm sorry to intervene again, but I think these descriptions do not accurately describe what's visible on the landing page. Please limit these descriptions to explain the various widgets visible on the landing page only. It's fine to then add a comment that further information is available when clicking on the link that is active on several of the widgets.

Author: I tried following your comment. Please take a look and let me know what you think.

Reviewer 2: Looking better, thanks! I left a number of textual improvement suggestions, please take them into consideration.
